src/JSDomEnvironment.js

```diff
@@ -105,7 +107,10 @@ JSDomEnvironment.prototype.dispose = function() {
 };
 
 JSDomEnvironment.prototype.runSourceText = function(sourceText, fileName) {
-  return this.global.run(sourceText, fileName);
+  var sandbox = vm.createContext(this.global);
```

Comment:
Eek no don't depend on _global... Seems better to create a <script> and insert it?